src/templates/ObservabilityPackDetails.js

```diff
@@ -32,6 +36,22 @@ const ObservabilityPackDetails = ({ data, location }) => {
       packId: pack.id,
     }
   );
+
+  const getPackUrl = () => {
+    const baseUrl =
+      'https://github.com/newrelic/newrelic-observability-packs/tree/main/packs';
+
+    if (!pack.logoUrl) {
+      return baseUrl;
+    }
+
+    /**
+     * logoUrl looks something like: 'https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/couchbase/logo.svg'
+     */
+    const [packName, ..._] = pack.logoUrl.split('/').slice(-2);
+    return join(baseUrl, packName);
```

Comment:
The `path` module is exclusively for system file paths. The equivalent for URLs would be
```suggestion
    return new URL(packName, baseUrl).href;
```

Reply:
swapped to url